src/assignments/coded-messages.rst

```diff
@@ -0,0 +1,336 @@
+
+   In this case, each original letter is shifted 2 places to the right in the alphabet.
+
+Open a new project in your code editor and complete the following tasks.
+
+The ``alphabet_position`` Function
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#. Define a function called ``alphabet_position`` with one parameter. The
+   parameter receives a string containing a single letter.
+#. The function returns the *index* of that letter within the alphabet.
+#. The function should be *case-insensitive*. Capital and lowercase letters
+   return the same result.
+
+   .. list-table:: Sample Results
+      :header-rows: 1
+
+      * - ``letter``
+        - Return Value
+      * - a or A
+        - 0
+      * - d or D
+        - 3
+      * - z or Z
+        - 25
+
+#. *Don't forget to test your function!* In the ``main()`` function, call
+   ``alphabet_position`` and send in different letters as arguments. Be sure to
+   print the returned result to check your code.
+
+.. admonition:: Tip
+
+   #. You can assume the argument sent to the function will be a single letter.
+      Do not worry about what to do with longer strings or non-letter
+      characters.
+   #. ``alphabet_position`` should NOT print anything. It needs to *return* a
+      value instead.
+
+The ``shift_character`` Function
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#. Define the function ``shift_character(char, shift)`` with two parameters.
+   ``char`` accepts a *character* (a string of length 1) and ``shift`` accepts
+   an integer value.
+#. If ``char`` is a letter, the function returns a different one. The new
+   letter will be ``shift`` places to the right in the alphabet.
+#. The case of the original letter and the returned value should match. For
+   example, ``shift_character("A", 1)`` returns ``"B"``, instead of ``"b"``.
+#. If ``char`` is NOT a letter, then the function returns the same character.
+
+   .. list-table:: Sample Results
+      :header-rows: 1
+
+      * - ``char``
+        - ``shift``
+        - Return Value
+      * - a
+        - 13
+        - n
+      * - D
+        - 4
+        - H
+      * - z
+        - 3
+        - c
+      * - J
+        - 30
+        - N
+      * - ?, 6, &, etc.
+        - (any integer)
+        - ?, 6, &, etc. (no change for non-letter characters)
+
+#. *Test your function!* In ``main()``, call ``shift_character`` and send in
+   different characters and shift values. Be sure to print the letter returned
+   to make sure your code works properly before moving on.
+
+.. admonition:: Tip
+
+   #. You should make use of your ``alphabet_position`` function! Thinking in
+      terms of index values helps.
+   #. Notice that the function "wraps around" when it reaches the end of the
+      alphabet. If a shift moves the program past the letter ``'z'`` (index
+      value ``25``), it moves back to the start of the alphabet and continues
+      counting with ``'a'`` (index value ``0``).
```

Comment:
Again to avoid the scare quotes, this can be reworded:

Notice in the sample values table that the function wraps around when it reaches the end of the alphabet. If a shift ...